pkgs/development/php-packages/phive/default.nix

```diff
@@ -0,0 +1,29 @@
+{ mkDerivation, fetchurl, makeWrapper, lib, php }:
+
+mkDerivation rec {
+  pname = "phive";
+  version = "0.15.0";
+
+  src = fetchurl {
+    url = "https://github.com/phar-io/phive/releases/download/${version}/phive-${version}.phar";
+    sha256 = "sha256-crMr8d5nsVt7+zQ5xPeph/JXmTEn6jJFVtp3mOgylB4=";
+  };
+
+  dontUnpack = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
```

Comment:
When overriding `installPhase`, you should also provide `runHook preInstall` and `runHook postInstall`. See <https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md> for more info.

Reply:
Indeed sorry about this, going to fix it asap. Bad copy paste bad !

Reply:
I added all the missing hooks in PHP Packages in the second commit. Let me know if it can be merged within this PR.